web/src/Update.elm

```diff
@@ -133,6 +133,21 @@ update msg model =
             in
             { model | democracies = Dict.insert democId setDelegateState model.democracies } ! []
 
+        Debug str ->
```

Comment:
Probs best to call this `AppendDebugLog` or something instead to be clearer. Or just `LogMsg` - then you can do other stuff with it too like add it to the UI somewhere (maybe have an alert) and log to console, etc.